addons/api/mirage/config.js

```diff
@@ -599,6 +600,48 @@ function routes() {
     return new Response(200);
   });
 
+  // app-tokens
+
+  this.post('/app-tokens', function ({ appTokens, scopes }) {
+    // Create new app token or handle clone
+    const attrs = this.normalizedRequestAttrs();
+
+    // Find the scope for the app token. Scope can be global, org or proj level.
+    const scope = scopes.find(attrs.scopeId);
+    if (!scope) {
+      return new Response(
+        404,
+        {},
+        {
+          errors: [
+            {
+              status: 404,
+              message: 'Scope not found',
+            },
+          ],
+        },
+      );
+    }
```

Comment:
Since this is mirage, lets make this always return a success. If we cannot find the scope from the `attrs.scopeId`, we can use `global` as a default.